File: AppGenerateFiles/dbo/Model/Transaction_Ingresos_Egresos.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Transaction_Ingresos_Egresos : EntityClass {
       [PrimaryKey(Identity = false)]
       public int? id_transaccion { get; set; }
       public int? id_tipo_transaccion { get; set; }
       public Double? monto_dolares { get; set; }
       public Double? tasa_cambio { get; set; }
       public Double? monto_total { get; set; }
       public string? descripcion { get; set; }
       public string? nombre { get; set; }
       public int? id_cuenta { get; set; }
       public int? que { get; set; }
       public DateTime? fecha_anulado { get; set; }
       public string? banco { get; set; }
       public string? estado { get; set; }
       public int? numero_original { get; set; }
       public DateTime? fecha { get; set; }
       [ManyToOne(TableName = "Catalogo_Cuentas", KeyColumn = "id_cuentas", ForeignKeyColumn = "id_cuenta")]
       public Catalogo_Cuentas? Catalogo_Cuentas { get; set; }
       [ManyToOne(TableName = "Catalogo_Tipo_Transaccion", KeyColumn = "id_tipo_transaccion", ForeignKeyColumn = "id_tipo_transaccion")]
       public Catalogo_Tipo_Transaccion? Catalogo_Tipo_Transaccion { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Transaction_Ingresos_Egresos.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Catalogo_Cuentas }  from './Catalogo_Cuentas.js'
import { Catalogo_Tipo_Transaccion }  from './Catalogo_Tipo_Transaccion.js'
class Transaction_Ingresos_Egresos extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ id_transaccion;
   /**@type {Number}*/ monto_dolares;
   /**@type {Number}*/ tasa_cambio;
   /**@type {Number}*/ monto_total;
   /**@type {String}*/ descripcion;
   /**@type {String}*/ nombre;
   /**@type {Number}*/ que;
   /**@type {Date}*/ fecha_anulado;
   /**@type {String}*/ banco;
   /**@type {String}*/ estado;
   /**@type {Number}*/ numero_original;
   /**@type {Date}*/ fecha;
   /**@type {Catalogo_Cuentas} ManyToOne*/ Catalogo_Cuentas;
   /**@type {Catalogo_Tipo_Transaccion} ManyToOne*/ Catalogo_Tipo_Transaccion;
}
export { Transaction_Ingresos_Egresos }
